app/Console/Commands/CleanSquawkAssignmentsHistory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Squawk\SquawkAssignmentsHistory;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class CleanSquawkAssignmentsHistory extends Command
+{
+    protected $signature = 'squawks:clean-history';
+
+    protected $description = 'Delete any squawk assignment history that is older than a specified age';
+
+    public function handle()
```

Comment:
void typehint because why not - unless the abstract doesn't likey of course

Reply:
Done